src/bug_report.rs

```diff
@@ -52,6 +52,15 @@ fn get_pkg_branch_tag() -> &'static str {
 }
 
 fn make_github_issue_link(environment: Environment) -> String {
+    let shell_syntax = match environment.shell_info.name.as_ref() {
+        "powershell" => "pwsh",
```

Comment:
This does not work for powershell core (or...whatever it's called? I'm not very familiar with the renamings around Core) because the binary name is "pwsh"
```suggestion
        "powershell" | "pwsh" => "pwsh",
```

Reply:
One minor change to account for a recent PR which added a new name for pwoershell core.

Reply:
Done.